docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,137 @@
+7. Click the "Pay this invoice" button in the email and follow the instructions to enter your payment details securely on Stripe website.
+8. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+   - Note that we don't send out a receipt. If you want to keep a receipt, click the "Download invoice" button after completing the payment on Stripe.
+
+.. image:: imgs/paid_api/stripe_email.png
+  :align: center
+  :alt: Stripe Email
+
+
+Using the API Key and URL
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. After your payment is processed, you will receive an email from ``Unstructured`` containing your unique API credentials and URL.
+2. Check your inbox (and the spam folder, just in case) for an email from our domain.
+3. Save these credentials securely, as you will need them to access and integrate our API with your applications.
+
+.. image:: imgs/paid_api/welcome_email.png
+  :align: center
+  :alt: Welcome Email
+
+
+SaaS API Account Maintenance
+-----------------------------
+
+Opt-in to Usage Emails
+^^^^^^^^^^^^^^^^^^^^^^
+
+To receive regular updates on usage statistics, please follow these steps:
+
+1. On `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage, navigate to FAQ section and click the "Subscribe to Usage Updates" link.
+2. Enter your email address in the provided field.
+3. Click the "Submit" button to complete the subscription process.
+
+You will receive a confirmation email shortly after submitting your information.
+
+.. image:: imgs/paid_api/opt-in.png
+  :align: center
+  :alt: Opt-in Email
+
+Update Payment Method
+^^^^^^^^^^^^^^^^^^^^^
+
+If you need to update your billing information, fill out the "Update Billing" form.
+
+1. On `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage, navigate to FAQ section and click the "Update Payment Method" link.
+2. Enter your email address in the provided field.
+3. Click the "Submit" button to complete the subscription process.
+
+Upon successful form submission, you will be redirected to the Stripe payment page. Follow the instructions to enter your payment details securely. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+You will receive a confirmation email about the new payment method.
+
+.. image:: imgs/paid_api/update_billing.png
+  :align: center
+  :alt: Update Billing
+
+Cancel Plan
+^^^^^^^^^^^
+
+To cancel your current plan, please complete the following steps:
+
+1. On `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage, navigate to FAQ section and click the "Cancel Plan" link.
+2. When the pop-up form appears, provide your registered *Email* in the first field.
+3. Enter your *Current API Key* in the following field.
```

Comment:
I'd change the italic _Current API Key_ and _Submit_ to bold. In the PDF the italic isn't very prominent.